docker-compose.yml

```
@@ -23,12 +23,64 @@ services:
     networks:
       - mynetwork
 
+  mysql-classroom:
+    image: mysql:8
+    container_name: mysql-classroom
+    restart: always
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: yes
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)